src/main/java/io/vertx/ext/sql/SQLConnection.java

```diff
@@ -34,6 +35,16 @@
 public interface SQLConnection extends AutoCloseable {
 
   /**
+   * Sets the desired options to be applied to the connection when a statement is executed.
```

Comment:
How about adding a comment to indicate if these options should be re-applied any time a connection is acquired from the pool?